lib/assessment.rb

```diff
@@ -34,7 +39,7 @@ def every_even(numbers)
   # Write ruby code that returns an array with all of the
   # entries from the words enumerable capitalized.
   def transform(words)
-    # Your code goes here
+    str[Array] = new str.upcase# Your code goes here
```

Comment:
When you want to perform a transformation on the elements of a group, the right tool is `map`.

`words.map {|word| word.capitalize}`
OR
`words.map(&:capitalize)`